include/knowhere/config.h

```diff
@@ -299,7 +299,7 @@ class Config {
                     *ptr->val = ptr->default_val.value();
                     continue;
                 }
-                if (!json[it.first].is_number_float()) {
+                if (!json[it.first].is_number()) {
```

Comment:
The arithmetic overflow check below becomes invalid, and all float values in the config can be float/int, is this acceptable? @liliu-z

Reply:
Looks good to me. cc @Presburger as the original author.

Reply:
Looks good.